client/mysqladmin.cc

```diff
@@ -1093,6 +1093,7 @@ static int execute_commands(MYSQL *mysql,int argc, char **argv)
       /* Warn about password being set in non ssl connection */
 #if defined(HAVE_OPENSSL) && !defined(EMBEDDED_LIBRARY)
       uint ssl_mode;
+      ssl_mode = 0;
```

Comment:
Likewise, and initialize at declaration

Reply:
I wanted to do that, but I couldn't - this part of the function is inside a goto jump (from a later part to an earlier label), and initializing within it is a compilation error.